### DISCUSSION TITLE
SortedObservableCollectionAdaptor Avoiding Replace

### DISCUSSION THREAD

**Discussion by ccopsey** [reactivemarbles/DynamicData#724] [Ideas]
Background: https://github.com/reactivemarbles/DynamicData/issues/640#issuecomment-1246421903

I just lost a day and half chasing down a very curious undesirable behaviour which I now realise was caused by the above.  It took me that long to realise where the problem lied, and from there I could search for info and solutions.

Is there a reason why we can't add `useReplaceForUpdates` as an argument to the `ISortedChangeSet` flavour of `Bind()` in a similar manner to how it is implemented in `ObservableCollectionAdaptor`?

I am guaranteed to forget this episode and end up scratching my head for another day at some point in the (not too distant) future.

I would be happy to propose the PR.

**Comment by glennawatson** [comment]
You can probably make a feature request to add extra overload method or new method. It's a bit different to the issue you referenced which was a bug report. Roland is more likely to approve the idea of it if it's new functionality and a new method or overload. 

That way in the issue you can coordinate how to design it.

**Reply by ccopsey** [reply]
In my mind it is an extra overload to `Bind()`, with an extra if-clause in `case ChangeReason.Update`.  Very simple.  But the amount of detail in that referenced comment, along with no apparent attempt to fix what looks to me to be simple, makes me question if I fully understand what I'm looking it.

I shall plough on and give it a go.
